src/applications/find-forms/api/index.js

```diff
@@ -26,11 +26,16 @@ export const fetchFormsApi = async (query, options = {}) => {
 
   const forms = response?.data;
   const onlyValidForms = forms?.filter(
-    form => form.attributes?.validPDF || form.attributes?.validPdf,
+    form =>
+      (form.attributes?.validPDF || form.attributes?.validPdf) &&
+      form.attributes?.deletedAt === null,
   );
 
+  // checks to see if all the forms in the results have been tombstone/ deleted.
+  const allFormsTombstone = forms?.length > 0 && onlyValidForms?.length === 0;
+
```

Comment:
I'm trying to wrap my head around the relationship between the case of a form having an invalid download URL (determined by "validPdf", which likely means there was a typo while inputting the PDF URL into the DB) versus a form being marked as deleted (which might happen if a bill was passed that retired a form.)

If I follow the logic...we consider "allFormsTombstone" if there are forms in the JSON response but none were considered valid either due to the "validPdf" property being false or "deleteAt" having a value that is not null. I feel like these should be accounted for separately?

"onlyValidForms" -> all forms in `response.data` that have `validPdf` => true 
"allFormsTombstone" -> true if _all_ forms in `response.data` have a value for `deletedAt`

You could totally be right in how you're doing it now though. I just want to make sure that someday if form 10-10EZ is searched for but the DB contains a typo for its PDF URL that we don't tell the user that it's been retired or deleted.

Reply:
Your synopsis of the purpose of onlyValidForms and allFormsTombstone are correct.